api/graphite.go

```diff
@@ -695,11 +695,22 @@ func (s *Server) executePlan(ctx context.Context, orgId uint32, plan expr.Plan)
 	reqs := NewReqMap()
 	metaTagEnrichmentData := make(map[string]tagquery.Tags)
 
+	// Map identical series expressions to reduce round trips
+	resolveSeriesRequests := make(map[expr.Req][]expr.Req)
+	for i, r := range plan.Reqs {
+		strippedreq := expr.Req{
```

Comment:
why are you stripping out the optimization flags and consolidator?

Reply:
They aren't relevant for uniqueness of resolving the series query into the matching series.

Reply:
right. can you add a comment to the code to clarify that

Reply:
Sure